### PR TITLE
Spawn points

### PR DESCRIPTION
**Pull request by niko227**

### CODE REVIEW

**Comment by sashadgt** [comment]
What are .mat for in this commit?

**Comment by Solero93** [comment]
There should only be 1 file changed, right @niko227 ?

**Review by Solero93 — DISMISSED** [review]
Test

File: Assets/Scripts/Model/Shop.cs [hunk 1]
```diff
@@ -62,6 +63,9 @@ public List<Unit> getPurchasableUnits(int numCoins)
     public void purchaseUnit(Unit unitToPurchase)
     {
         // TODO Missing how to spawn it somewhere
```
Comment (Solero93) [inline Assets/Scripts/Model/Shop.cs:65]: Have you tried this out on the scene?

**Review by Solero93 — COMMENTED** [review]
To test

**Pull request closed without merging**